documentation/src/components/side-nav.ts

```diff
@@ -16,7 +16,7 @@ import {
     property,
     PropertyValues,
 } from 'lit-element';
-import { ComponentDocs } from '../../components';
+import { ComponentApiDocs } from '../../api-docs';
```

Comment:
This is the smaller of the two files, and has the data in the same structure so it will serve as a drop-in replacement here.